config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-v1beta1-release-1.12.yaml

```diff
@@ -1,9 +1,9 @@
 periodics:
-- name: periodic-cluster-api-provider-azure-capi-e2e-v1beta1-release-1-11
+- name: periodic-cluster-api-provider-azure-capi-e2e-v1beta1-release-1-12
   decorate: true
   decoration_config:
     timeout: 4h
-  interval: 72h
+  interval: 24h
```

Comment:
Why are these intervals being decreased?

Reply:
These intervals seemed to be exclusive to release-1.11 jobs so I reverted them to defaults when porting to release-1.12. 
I am not sure if I should be keeping them to `72h`